rca/static_src/javascript/programmes/components/ProgrammesCategories/ProgrammesCategories.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { useLocation } from 'react-use';
+
+import { programmeCategories } from '../../programmes.types';
+import { searchProgrammes } from '../../programmes.slice';
+
+import CategoriesTablist from './CategoriesTablist';
+import CategoriesPanels from './CategoriesPanels';
+
+/**
+ * A list of programmes matching a search or filter.
+ * The list auto-magically appears when matches are found.
+ */
+const ProgrammesCategories = ({ categories, applyFilter }) => {
+    const loc = useLocation();
+    const activeCategory = loc.hash.replace('#', '') || categories[0].id;
```

Comment:
Feels a bit unsafe to treat any hash as a category – but I’ll switch this over to use query parameters in subsequent work.